pennylane/ops/qubit.py

```diff
@@ -1602,6 +1686,9 @@ def _matrix(cls, *params):
 
         return U
 
+    def adjoint(self, do_queue=False):
+        return QubitUnitary(self.data[0].conj().T, wires=self.wires, do_queue=do_queue)
```

Comment:
This will only work for jax/autograd/numpy, but fail for TF and Torch! We need a `qml.math.conj()` to go alongside `qml.transpose`.

Reply:
Ooof, good to know...

Reply:
Maybe worth opening an issue for this. I think it is too out of scope to include here, but I don't want to forget to fix this either.

Reply:
Added an issue https://github.com/PennyLaneAI/pennylane/issues/1118